cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -30,6 +30,153 @@ Cask Data Application Platform Release Notes
    :backlinks: none
    :depth: 2
 
+`Release 5.1.0 <http://docs.cask.co/cdap/5.1.0/index.html>`__
+=============================================================
+
```

Comment:
Can we break this into sections. Suggested breakdowns

New Features
Improvements
Deprecations
Bug Fixes

Reply:
@sreevatsanraman I remember the script used to do it based on the type of jira... not sure what happened.

Reply:
I have never used it. If you have the script do share. Otherwise it should be easy to organize this into buckets.

Reply:
I don't have a different script. Just that whenever I've reviewed in previous releases, its already been grouped by the jira type. Wonder what happened.